src/status_im2/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -0,0 +1,199 @@
+  [theme]
+  [rn/view
+   {:style {:padding-horizontal 20
+            :padding-bottom     16}}
+   [quo/text
+    {:size                :paragraph-2
+     :weight              :medium
+     :style               (style/section-label theme)
+     :accessibility-label :summary-from-label}
+    (i18n/label :t/details)]
+   [rn/view
+    {:style (style/details-container theme)}
+    [quo/data-item
+     {:container-style {:flex   1
+                        :height 36}
+      :blur?           false
+      :description     :default
+      :icon-right?     false
+      :card?           false
+      :label           :none
+      :status          :default
+      :size            :small
+      :title           (i18n/label :t/est-time)
+      :subtitle        "3-5 min"}]
+    [quo/data-item
+     {:container-style {:flex   1
+                        :height 36}
+      :blur?           false
+      :description     :default
+      :icon-right?     false
+      :card?           false
+      :label           :none
+      :status          :default
+      :size            :small
+      :title           (i18n/label :t/max-fees)
+      :subtitle        "€188,70"}]
+    [quo/data-item
+     {:container-style {:flex   1
+                        :height 36}
+      :blur?           false
+      :description     :default
+      :icon-right?     false
+      :card?           false
+      :label           :none
+      :status          :default
+      :size            :small
+      :title           (i18n/label :t/user-gets {:name "Mark"})
+      :subtitle        "149.99 ETH"}]]])
+
+(defn- f-view-internal
+  [theme]
+  (let [reset-slider?        (reagent/atom false)
+        margin-top           (safe-area/get-top)
+        biometric-auth?      true
+        on-close             #(rf/dispatch [:navigate-back-within-stack :wallet-select-asset])
+        status-account-props {:customization-color :purple
+                              :size                32
+                              :emoji               "🍑"
+                              :type                :default
+                              :name                "Collectibles vault"
+                              :address             "0x0ah...78b"}
+        user-props           {:full-name           "M L"
+                              :status-indicator?   false
+                              :size                :small
+                              :ring-background     (resources/get-mock-image :ring)
+                              :customization-color :blue
+                              :name                "Mark Libot"
+                              :address             "0x0ah...78b"
+                              :status-account      (merge status-account-props
+                                                          {:size  16
+                                                           :name  "New house"
+                                                           :emoji "🍔"})}]
+    (fn []
```

Comment:
If theme changes, this view won't rerender, si ce we should receive `them` also in the `fn []` at the end